packages/paste-website/src/components/tokens-list/utils/sentenceCase.ts

```diff
@@ -0,0 +1,8 @@
+export const sentenceCase = (catName: string): string => {
```

Comment:
todo: Fun fact, we have duplicates of this type of utility. The other doesn't handle kebab case, and this one does. We should consider at least co-locating them and renaming to be explicit about what cases they handle (since the types don't indicate anything about these specifics).